### PR TITLE
docs(#16): Add information about how to create the website infrastructure

### PR DESCRIPTION
**Pull request by bra1n2h**
## Description
Adding information how to create the website infrastructure.

## Tasks
- [ ] fix README.md file

## Acceptance Criteria
- [ ] README.md file fixed

## Summary by CodeRabbit

- **Documentation**
	- Enhanced instructions for deploying infrastructure and running pipelines in AWS.
	- Added guidance for creating website infrastructure post-deployment using `terraspace all up`.
	- Expanded CLI instructions for using the `make` command to trigger pipelines.
	- Updated AWS Management Console section with a step-by-step guide for CodePipeline releases.
	- Introduced a new entry for `trigger-pipeline` in the `make` command options.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -161,6 +161,8 @@ After you configured everything you can deploy infrastructure by running such co
 - `make terraspace-all-validate`
 - `make terraspace-all-up`
 
+After you deployed infrastructure you can trigger CI/CD for website setup by running such command.
+- `terraspace all up`
```
Comment (Kravalg) [inline README.md:165]: It is not a trigger for the pipeline, `terraspace all up` will create the website infrastructure itself
Please be more concrete and specific with the command and change it

**Review by Kravalg — CHANGES_REQUESTED** [review]
Let’s change the cli command for pipeline trigger

**Review by Kravalg — CHANGES_REQUESTED** [review]
Could you please include detailed instructions on how to set up and run the changes in AWS? Specifically, we need:

CLI Instructions: Step-by-step commands using AWS CLI for setting up and initiating the CodePipeline, ensuring all parameters and necessary configurations are clearly defined.
AWS Management Console Instructions: A guide for manually setting up the pipeline through the AWS Management Console, including screenshots or detailed descriptions of each step, if possible.
These instructions are crucial for ensuring that the team can efficiently test and deploy the pipeline changes in a consistent and controlled manne

**Comment by bra1n2h** [comment]
> Could you please include detailed instructions on how to set up and run the changes in AWS? Specifically, we need:
> 
> CLI Instructions: Step-by-step commands using AWS CLI for setting up and initiating the CodePipeline, ensuring all parameters and necessary configurations are clearly defined. AWS Management Console Instructions: A guide for manually setting up the pipeline through the AWS Management Console, including screenshots or detailed descriptions of each step, if possible. These instructions are crucial for ensuring that the team can efficiently test and deploy the pipeline changes in a consistent and controlled manne

I would like to clarify that I understood you correctly, add instructions for running changes in CodePipeline using cli and aws management console, but before that, based on the previous request for changes in the Makefile, add a command to run changes via cli?

**Comment by Kravalg** [comment]
> > Could you please include detailed instructions on how to set up and run the changes in AWS? Specifically, we need:
> > CLI Instructions: Step-by-step commands using AWS CLI for setting up and initiating the CodePipeline, ensuring all parameters and necessary configurations are clearly defined. AWS Management Console Instructions: A guide for manually setting up the pipeline through the AWS Management Console, including screenshots or detailed descriptions of each step, if possible. These instructions are crucial for ensuring that the team can efficiently test and deploy the pipeline changes in a consistent and controlled manne
> 
> I would like to clarify that I understood you correctly, add instructions for running changes in CodePipeline using cli and aws management console, but before that, based on the previous request for changes in the Makefile, add a command to run changes via cli?

Yes, you've got it exactly right
Please add the CLI instructions for running the changes in CodePipeline, and also include the instructions for setting up and managing the pipeline through the AWS Management Console
Additionally, based on our earlier discussions about the Makefile updates, please integrate a command to facilitate these changes via the CLI

**Review by Kravalg — APPROVED** [review]

**Pull request merged**